AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java

```diff
@@ -81,7 +81,7 @@
     private static final int[] FACTOR_ADDITION_VALUES = { -150, 0, 150 };
     public static final int RESCHEDULE_FACTOR = Consts.STARTING_FACTOR;
 
-    private String mName = "std2";
+    protected String mName = "std2";
```

Comment:
Or my suggestion in the comments to set this in SchedV1 constructor
```suggestion
    private String mName = "std2";
```

Reply:
Corrected. Thanks